src/main/java/com/thebois/models/Position.java

```diff
@@ -96,4 +101,21 @@ public Position deepClone() {
         return new Position(this.posX, this.posY);
     }
 
+    /**
+     * Calculates the Manhattan distance to the destination.
+     *
+     * <p>
+     * Manhattan distance is distance calculated through a grid. Doesn't allow diagonal movements.
+     * </p>
+     *
+     * @param destination The position to check the distance to.
+     *
+     * @return The manhattan distance to the destination.
+     */
+    public int manhattanDistanceTo(final Position destination) {
+        final float distanceX = Math.abs(getPosX() - destination.getPosX());
+        final float distanceY = Math.abs(getPosY() - destination.getPosY());
+        return (int) (distanceX + distanceY);
+    }
```

Comment:
I know everything is done with whole numbers, but the return value should probably be rounded